fromscratchtoml/test/naive_bayes/test_gaussian_nb.py

```diff
@@ -0,0 +1,32 @@
+
+# coding: utf-8
+
+# In[10]:
+
+
+from fromscratchtoml.naive_bayes import GaussianNB
+from unittest import TestCase
+import numpy as np
+
+
+# In[9]:
+
+
+def TestGaussianNB(TestCase):
+    def testPredictions(self):
+        my_data = np.genfromtxt('data.csv', delimiter=',',dtype=str)
```

Comment:
Please check how we are getting data files in other tests and move this test file to `test_data` folder. Also give it a appropriate name.